kong/core/handler.lua

```diff
@@ -118,16 +125,10 @@ return {
           "' with: "..tostring(err))
       end
 
-      -- if set `host_header` is the original header to be preserved
-      var.upstream_host = host_header or 
+      -- if set, `upstream_host` is the original header to be preserved
```

Comment:
this comment should be `ctx.upstream_host` because the line below has two references